tests/pt-br/cover_HassTurnOn.yaml

```diff
@@ -1,2 +1,25 @@
 language: pt-br
-tests: []
+tests:
+  - sentences:
+      - abra a porta da garagem
+      - abrir a porta da garagem
+    intent:
+      name: HassTurnOn
+      slots:
+        device_class: garage
+        domain: cover
+    response: Aberto
+
+  - sentences:
+      - abra as cortinas no cozinha
+      - abrir as cortinas no cozinha
```

Comment:
Could you adjust the sentence to "abrir as cortinas da cozinha"

Reply:
ok